### PR TITLE
Make the add post survey list accessible

### PR DESCRIPTION
**Pull request by renujain31**
This pull request makes the following changes:
- It makes the add post mainsheet window accessible with keyboard.

Testing checklist:
- [ ] Go to /views/map.
- [ ] Click on the add post button.
- [ ] The mainsheet container is accessible via keyboard.

- [x] I certify that I ran my checklist

Recording:
add_post_menu

Fixes ushahidi/platform#4188 .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by Angamanga** [comment]
@renujain31 I would like to change the styling a bit here and move the close-button to the right, to make it similar to the other modal-windows. Could you look into that in the pattern-library? I am thinking something like this:
Screenshot 2021-01-26 at 13 37 33
Thank you 🙏

**Comment by Angamanga** [comment]
@renujain31 Or maybe you did that already? It looks like it on your recording? When I check locally it looks like this:
Screenshot 2021-01-26 at 13 39 46

File: app/common/directives/mainsheet-container.directive.js [hunk 1]
```diff
@@ -52,15 +53,26 @@ function MainsheetContainer($rootScope, $compile, MainsheetService) {
             mainsheetContent.html(template);
             $compile(mainsheetContent)(templateScope);
 
+            $timeout(function () {
```
Comment (Angamanga) [inline app/common/directives/mainsheet-container.directive.js:56]: Lets wait for Rominas review on my change here: https://github.com/ushahidi/platform-client/pull/1637#discussion_r564068092, and if its ok lets do the same approach to the mainsheet :)

**Comment by renujain31** [comment]
Hey, I had already added that in platform pattern library but forgot to make the pull request. Here it is: https://github.com/ushahidi/platform-pattern-library/pull/268

**Review by Angamanga — APPROVED** [review]
Works fine now 🎉 Approved!

**Comment by Angamanga** [comment]
@Obadha2 Ready for QA!

**Comment by Obadha2** [comment]
QA'd, passes. 👍

**Pull request merged**